### ISSUE TITLE
[semicolon] false positive lint error reported for ES6 module syntax

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#441]
on v2.3.0-beta, reported by @vanko-dev

```ts
import {Router} from 'aurelia-router';
```

tsling messages:
error app.ts[6, 33]: missing semicolon
error app.ts[7, 15]: missing semicolon

**Comment by leeavital** [comment]
I haven't been able to reproduce this on master, but I put in a PR for some regression tests that make sure this doesn't back.

**Comment by vanko-dev** [comment]
I was launching tslint via gulp-tslint. It used not last version of tslint. Sorry for confusion. Tslint works good directly in my case.

**Comment by adidahiya** [comment]
thanks for the update
